### DISCUSSION TITLE
BT-150 validation error in notices 36 and 37

### DISCUSSION THREAD

**Discussion by EEanalyst** [OP-TED/eForms-SDK#334] [Q&A] [answered]
BT-150 is forbidden in notice types 36 and 37 but if we don't add it into XML we will get an error: 
Contract Identifier (BT-150) value must be unique.

We are developing on version 1.3.2. Is it solved in later versions? Should there be a contract identifier in 36 and 37 or not?

**Comment by YvesJo** [comment, marked as answer]
Hi,
This issue has been fixed in later versions with the rule context applied at the field level directly so that the check for unicity does not conflicts with the existence requirements and only gets triggered when the field exists.
KR.
